Makefile

```diff
@@ -1,30 +1,14 @@
-SHELL := /bin/bash
+include .bingo/Variables.mk
 
-BIN      	= $(CURDIR)/bin
-BUILD_DIR   = $(CURDIR)/build
+SHELL := /bin/bash
 
-GOPATH		= $(HOME)/go
-GOBIN		= $(GOPATH)/bin
-GO			?= GOGC=off $(shell which go)
+GO	?= GOGC=off $(shell which go)
 
 # Printing
 V = 0
 Q = $(if $(filter 1,$V),,@)
 M = $(shell printf "\033[34;1m▶\033[0m")
 
-# Tools
-$(BUILD_DIR):
-	@mkdir -p $@
```

Comment:
Could you please update the Makefile to make sure that the bingo dependencies are available before running commands such as `make generate`?

Reply:
As of now i don't see a `generate` task in the `Makefile` even in the [main branch](https://github.com/jeroenrinzema/psql-wire/blob/main/Makefile). 

If i am getting it right, do you want the binary dependencies to be present before running any make command?

Reply:
The default behavior of bingo is "lazy installation" i.e. it is installed just before it is used, for example if you see the below screenshot i have accessed `$(GOLANGCI_LINT)` in 2 places one in the dependency section and other in the task body. The dependency section one installs the golangci-lint if there are changes to the module file of golangci-lint

## In projects Make file
<img width="743" alt="image" src="https://github.com/jeroenrinzema/psql-wire/assets/763760/33adc65a-f020-4e39-8d93-e482b15287e3">

## In Variables.mk file

<img width="1439" alt="image" src="https://github.com/jeroenrinzema/psql-wire/assets/763760/f8a7967f-1431-4e23-afd0-894bb445e668">